external-applications/quarkus-workshop-super-heroes/pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus.ts.openshift</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+
+    <artifactId>quarkus-workshop-super-heroes</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Quarkus OpenShift TS: Workshop Super Heroes</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy</artifactId>
+        </dependency>
```

Comment:
Do you need this dep?

Reply:
I am just using the constants from WS API jar, I should probably just use that actually

Reply:
Ah OK, no, that makes sense.

Reply:
Perhaps just add it to `<scope>test</scope>`.

Reply:
https://github.com/quarkusio/quarkus/blob/master/bom/runtime/pom.xml#L1612
You can add:
```xml
            <dependency>
                <groupId>org.jboss.spec.javax.ws.rs</groupId>
                <artifactId>jboss-jaxrs-api_2.1_spec</artifactId>
                <scope>test</scope>
            </dependency>
```

Reply:
Fixed.